src/effects/presets/effectchainpresetmanager.cpp

```diff
@@ -857,8 +858,31 @@ EffectsXmlData EffectChainPresetManager::readEffectsXml(
         }
     }
 
-    return EffectsXmlData{
-            eqEffectManifests, quickEffectPresets, standardEffectChainPresets, mainEqPreset};
+    // Read names of presets that were loaded into stem QuickEffects on last shutdown
+    QDomElement quickStemEffectPresetsElement =
+            XmlParse::selectElement(root, EffectXml::kQuickStemEffectList);
+    const QDomNodeList quickStemEffectNodeList =
+            quickStemEffectPresetsElement.elementsByTagName(
+                    EffectXml::kChainPresetName);
+    quickStemEffectPresets.reserve(quickStemEffectNodeList.count());
+    for (const auto& qseNode : quickStemEffectNodeList) {
```

Comment:
I also suffer this compiler error. qseNode is not used.
```suggestion
    for (for (int i = 0; i < quickStemEffectNodeList.size(); ++i) {
```